pennylane_ionq/device.py

```diff
@@ -132,9 +155,12 @@ def operations(self):
     def apply(self, operations, **kwargs):
         self.reset()
         rotations = kwargs.pop("rotations", [])
+        self.error_mitigation = kwargs.pop("error_mitigation", {})

```

Comment:
might this already be set from device initialization?

Reply:
yup! thanks for the catch

Reply:
do you also want to add support for it being passed to `apply` through kwargs, or just remove this line?

Reply:
probably just remove it